.gitignore

```diff
@@ -1,11 +1,11 @@
-__pycache__
-*.egg-info
-*~
-*.bak
-.ipynb_checkpoints
-.tox
-.DS_Store
-\#*#
-.#*
-.coverage
-*.swp
\ No newline at end of file
+-__pycache__        
```

Comment:
There's some copy paste error here. It seems that the '-' at the beginning of each line-diff has been inserted as actual text in the doc. These lines should not begin with -. Also, it seems they now all end with extra spaces.